codegenerator/cli/templates/static/codegen/src/db/DbFunctions.res

```diff
@@ -98,10 +98,10 @@ module ChainMetadata = {
     @as("chain_id") chainId: int,
     @as("block_height") blockHeight: int,
     @as("start_block") startBlock: int,
-    @as("end_block") endBlock: option<int>,
-    @as("first_event_block_number") firstEventBlockNumber: option<int>,
-    @as("latest_processed_block") latestProcessedBlock: option<int>,
-    @as("num_events_processed") numEventsProcessed: option<int>,
+    @as("end_block") endBlock: Js.Nullable.t<int>,
```

Comment:
```suggestion
    // The values below could use `Js.Null.t` instead of `Js.Nullable.t`
    // It just needs to be confiremed that the postgres lib never returns
    // undefined.
    @as("end_block") endBlock: Js.Nullable.t<int>,
```